pkg/search/backendstore/opensearch.go

```diff
@@ -232,27 +236,31 @@ func (os *OpenSearch) indexName(us *unstructured.Unstructured) (string, error) {
 	os.l.Lock()
 	defer os.l.Unlock()
 
-	if _, ok := os.indices[name]; !ok {
```

Comment:
The mapping settings before the modification did not take effect, and dynamic mapping was used.